diff --git a/tensorflow/tensorflow.bzl b/tensorflow/tensorflow.bzl
index db310a5..0f21e4f 100644
--- a/tensorflow/tensorflow.bzl
+++ b/tensorflow/tensorflow.bzl
@@ -312,7 +312,7 @@ def _cuda_copts():
         "//third_party/gpus/cuda:using_nvcc": (
             common_cuda_opts +
             [
-                "-nvcc_options=expt-relaxed-constexpr",
+                "-nvcc_options=relaxed-constexpr",
                 "-nvcc_options=ftz=true",
             ]
         ),
diff --git a/third_party/gpus/crosstool/CROSSTOOL b/third_party/gpus/crosstool/CROSSTOOL
index 62799da..8db81a9 100644
--- a/third_party/gpus/crosstool/CROSSTOOL
+++ b/third_party/gpus/crosstool/CROSSTOOL
@@ -149,8 +149,6 @@ toolchain {
     linker_flag: "-Wl,--gc-sections"
   }
   linking_mode_flags { mode: DYNAMIC }
-
-  cxx_builtin_include_directory: "/usr/local/cuda-8.0/include"
 }
 
 toolchain {
@@ -244,4 +242,3 @@ toolchain {
     compiler_flag: "-fdata-sections"
   }
 }
-
